build.gradle

```diff
@@ -1,163 +1,37 @@
 plugins {
     id 'java-library'
-    id 'signing'
-    id 'maven-publish'
-    id "io.github.gradle-nexus.publish-plugin" version "1.0.0"
-    id "org.owasp.dependencycheck" version "8.1.0"
+    id "com.exactpro.th2.gradle.publish" version "0.2.0"
 }
 
 group = 'com.exactpro.th2'
 version = release_version
 
-ext {
-    sharedDir                   = file("${project.rootDir}/shared")
-}
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
issue: The gradle plugin has `th2JavaRelease` extension that should be used to configure the target java version

```kotlin
th2JavaRelease {
  targetJavaVersion.set(JavaVersion.VERSION_1_8)
}
```

Reply:
Could you please take a look on https://github.com/th2-net/th2-gradle-plugin/pull/108